packages/formatters/src/index.ts

```diff
@@ -4,6 +4,7 @@ export * from './junit';
 export * from './html';
 export * from './text';
 export * from './teamcity';
+export * from './github-actions';
```

Comment:
I don't think we can export it from this file. We use `process.cwd` which is specific to Node.js.
We may need to do something similar to what we did with `pretty` (at the very bottom of the file).
Could we also update README?

Reply:
Fixed in [d79e4b9](https://github.com/stoplightio/spectral/pull/2508/commits/d79e4b986353bb0092644ec6eb10b5c471e21f51) and [00b5dfc](https://github.com/stoplightio/spectral/pull/2508/commits/00b5dfc8f04d4b0c398baf8856d8debeb93d6c47).